07/ID_tryExcept

```diff
@@ -0,0 +1,52 @@
+# Input ID number in Czech Republic
+# write ouf corect/incorect ID
+
+
+# MONTHS 30 DAYS
+months_30 = [4, 6, 9, 11]
+
+# MONTHS 31 DAYS
+months_31 = [1, 3, 5, 7, 8, 10, 12]
+
+# YEARS
+years = list(range(86, 100)) + list(range(00, 23))
+ID_only_numbers = []
+
+while True:
+
+    ID = input('Napiš rodné číslo: ')
+    if ID[6] == "/":
+        try:
+            a = int(ID[:6])
+            b = int(ID[7:])
+        except ValueError:
+            print("v rodném čísle jsou nepovolené znaky")
+            continue
+    else:
+        print("lomítky chybí nebo je na špatném místě")
+        continue
+
+    ID_only_numbers = ID[:6] + ID[7:]
+    modulo_eleven = int(ID_only_numbers)
+    year_ID = int(ID_only_numbers[:2])
+    month_ID = int(ID_only_numbers[2:4])
+    day_ID = int(ID_only_numbers[4:6])
+    end = int(ID_only_numbers[6:])
+
+    if (modulo_eleven % 11 == 0) and (year_ID in years) and (end < 6000):
+
+        if month_ID > 50:
```

Comment:
Výhoda toho modula je, že ho môžeš použiť aj na tie mesiace ktoré po vydelení daným číslom vrátia 0 a ten zvyšok. Tebe ide o ten zvyšok a nepotrebuješ riešiť či to číslo reprezentujúce mesiac je alebo nie je väčšie ako 50. Takže r. 38 a 39 nepotrebuješ a stačí ak na riadkoch 41, 44 a 47 upravíš napr. v tom prvom prípade `month_ID == 2` na `month_id % 50 == 2`.